napari_allencell_annotator/view/images_view.py

```diff
@@ -349,6 +353,8 @@ def clear_all(self) -> None:
         Clear all image data from the model and the file widget.
         """
         self._annotator_model.clear_all_images()  # clear model
+        self.viewer.clear_layers()
+        self.file_widget.setCurrentItem(None)
```

Comment:
can these trigger in their own class on image_count_changed instead?

Reply:
refactored in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/2610531d02a28833a63da4f9752b24363394677c